ccp/src/main/res/values/strings.xml

```diff
@@ -472,7 +472,7 @@
     <string name="israil_hint" translatable="false">03-123-4567</string>
     <string name="ireland_hint" translatable="false">087 123 4567</string>
     <string name="indonesia_hint" translatable="false">21 8379 6236</string>
-    <string name="india_hint" translatable="false">241 123 5</string>
+    <string name="india_hint" translatable="false">90323 12345</string>
```

Comment:
How about instead something like below. I'm not a fan of how there are actual numbers here it seems confusing / maybe abusive if someone actually has the number
```suggestion
    <string name="india_hint" translatable="false">xxxxx nnnnn</string>
```